### DISCUSSION TITLE
Install extensions via Rigtools.

### DISCUSSION THREAD

**Discussion by prjamming** [3kh0/ext-remover#1271] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I was wondering if it is possible to use Rigtools to install extensions. I have an extension allowlist, blocklist, and forcelist. I'm on v125.

**Comment by no-name-6655** [comment]
Yes if you have the rigtools page open just press Ctrl+w it will open the extension installer

**Reply by prjamming** [reply]
No, that's how you close a tab.

**Reply by schoolexploitkid** [reply]
stop trolling

**Reply by akabutnicer** [reply]
> Yes if you have the rigtools page open just press Ctrl+w it will open the extension installer

ah ok I knew it!

**Comment by schoolexploitkid** [comment]
you cannot install blocked extensions via rigtools

**Reply by prjamming** [reply]
By that, do you mean it is blocked in the blocklist or that I am unable to install from the web store. There are only a handful of extension ids in the blocklist. Am I able to install extensions that aren't in the blocklist via Rigtools?

**Reply by S-PScripts** [reply]
im pretty sure you can only install extensions in the allowlist (https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/ExtensionInstallAllowlist%20Opener)

not anything else
